test/collectors_test.lua

```diff
@@ -6,29 +6,29 @@ local g = t.group('collectors')
 local metrics = require('metrics')
 local utils = require('test.utils')
 
-g.before_all = function()
+g.before_all( function()
     box.cfg{}
     box.schema.user.grant(
         'guest', 'read,write,execute', 'universe', nil, {if_not_exists = true}
     )
-end
 
-g.setup = function()
     -- Delete all previous collectors and global labels
     metrics.clear()
-end
+end)
 
-local function ensure_throws(desc, fun)
-   local ok, _ = pcall(fun)
-   t.assertTrue(not ok, desc .. ' throws')
-end
+g.after_each( function()
+    -- Delete all collectors and global labels
+    metrics.clear()
+end)
 
 g.test_counter = function()
-    ensure_throws('metrics.counter() w/o parameters', function()
+    -- metrics.counter() w/o parameters
+    t.assert_error( function()
```

Comment:
assert_error_msg_contains is usually self-explanatory

Reply:
Replaced `assert_error` with `assert_error_msg_contains`. It also helped me to find the bug in the global labels verification error, thanks! (Fixed it in another commit of this MR.)